Makefile

```diff
@@ -1,13 +1,17 @@
-BINARY_NAME=cmd/main.out
+BINARY_NAME?=cmd/main.out
 SOURCE_DIR=pkg/infra
+TEAM?=default
+LDFLAGS=-ldflags="-X 'infra/game/stages.mode=${TEAM}'"
+
+PACKAGES=$(shell go list ./pkg/infra | grep -v 'tests')
 
 PACKAGES=$(shell go list ./pkg/infra | grep -v 'tests')
 
 all: run
 
 build:
 		cd ${SOURCE_DIR}; go mod tidy
-		go build -o ${BINARY_NAME} infra
+		go build $(LDFLAGS) -o ${BINARY_NAME} infra 
```

Comment:
Are those changes necessary?

Reply:
I haven't made any person changes on these. I think they were added after a merge with the latest infra changes.